.github/workflows/all-checks.yml

```diff
@@ -0,0 +1,18 @@
+name: All checks pass
```

Comment:
Could this be named `Required`, since we might have checks that we would like to exclude at some point.